src/api_authentication_app_skeleton/src/actions/mixins/api/auth/skip_require_auth_token.cr

```diff
@@ -3,7 +3,7 @@ module Api::Auth::SkipRequireAuthToken
     skip require_auth_token
   end
 
-  # Since sign in is not required, current_user might be nil
+  # Since log in is not required, current_user might be nil
```

Comment:
Maybe `# Since authentication is not required, `? or `Since login is not required`